cmd/tfimport/main.go

```diff
@@ -150,5 +163,15 @@ func run() error {
 		return fmt.Errorf("failed to import %v resources:\n%v", len(errs), strings.Join(errs, "\n"))
 	}
 
+	if *dryRun && len(importCmds) > 0 {
+		log.Printf("Import commands:")
+		fmt.Printf("cd %v\n", *inputDir)
+		// The last arg in import could be several space-separated strings. These need to be quoted together.
+		for _, c := range importCmds {
+			args := strings.Split(c, " ")
+			fmt.Printf("%v %v %v %q\n", args[0], args[1], args[2], strings.Join(args[3:], " "))
```

Comment:
What's the difference with this vs just doing fmt.Printf(args...)

Reply:
Quoting the last part of the command is an importer-specific thing. The `Dry` runner shouldn't always quote the last args, because it won't always be called with an import command. Technically it should quote every part of cmd.Args, but that makes the output look ugly (`"terraform" "import" "google_project" "word1 word2 word3"`).

For whatever reason, `cmd.String()` returns everything unquoted and explicitly states that it's not suitable as input to a shell.